src/controllers/GraphHelpers.tsx

```diff
@@ -144,6 +145,7 @@ export const transformNodesToGQL = (nodes: any) => {
         delete gqlNode.parameters;
         delete gqlNode.description;
         delete gqlNode.paramGroups;
+        delete gqlNode.price;
```

Comment:
Maybe we should modify the backend to also accept a price in the node input; that way, the job/workflow will have a record of the price that was set on the service at the time of job submission.

(There is a longer story behind this - the short version is that the current architecture requires that the workflow nodes snapshot all the service parameters and data on job submission, so that if the service gets edited later on, the damplab admins can still see what parameters the client saw/entered before. If this is the operating logic, I am merely extending this logic to service prices.)
^ this is more a note to Future Us than to you :)